node/tessel-export.js

```diff
@@ -969,11 +971,16 @@ Tessel.SPI = function(params, port) {
 };
 
 Tessel.SPI.prototype.send = function(data, callback) {
-  this._port.cork();
+
   this.chipSelect.low();
-  this._port._tx(data, callback);
-  this.chipSelect.high();
-  this._port.uncork();
+
+  setTimeout(() => {
```

Comment:
ideally you would only uncork if you wanted a delay, otherwise you're going to get 100us-1ms of delay whether you like it or not.